server/src/test/java/org/elasticsearch/index/shard/RefreshListenersTests.java

```diff
@@ -324,9 +324,9 @@ public void testLotsOfThreads() throws Exception {
                         try (Engine.GetResult getResult = engine.get(get, engine::acquireSearcher)) {
                             assertTrue("document not found", getResult.exists());
                             assertEquals(iteration, getResult.version());
-                            SingleFieldsVisitor visitor = new SingleFieldsVisitor("test");
+                            DocumentStoredFieldVisitor visitor = new DocumentStoredFieldVisitor();
                             getResult.docIdAndVersion().reader.document(getResult.docIdAndVersion().docId, visitor);
-                            assertEquals(Arrays.asList(testFieldValue), visitor.fields().get("test"));
```

Comment:
I suspect I could just call `document(getResult.docIdAndVersion())` here and save some ceremony.